krcal/map_builder/map_builder_functions.py

```diff
@@ -563,6 +563,36 @@ def add_krevol(maps         : ASectorMap,
 
     return
 
+def check_drift_v_computation(drift_v      : np.ndarray ,
+                              dv_maxFailed : float = 0.5,
+                              **kwargs                  ) -> None:
+    """
+    Checks if drift velocity computation has failed in too many temporal
+    bins. If that is the case, map computation will abort.
+
+    Parameters
+    ---------
+    drift_v: np.ndarray
+        Array of drift velocities per time bin
+    dv_maxFailed: float
+        Threshold to consider too many failing fits.
+        (0.5 means that map creation will abort if more than 50% of fits fail)
+
+    Returns
+    ---------
+    Nothing
+    """
+
+    n_failing_dv_fits = sum(np.isnan(drift_v))
+    n_temp_bins       = len(drift_v)
+    fail_fits_ratio   = n_failing_dv_fits/n_temp_bins
+    message           = f'Drift_v fit failed in more than {dv_maxFailed*100}% of time bins.'
+
+    check_if_values_in_interval(values          = np.array(fail_fits_ratio),
+                                low_lim         = 0.                       ,
```

Comment:
Just being picky, but I think that for clarity, it's better you do the array conversion outside the calling

Reply:
or not, not so sure

Reply:
Okey, I did that because the other times we called this function we do it this way. But it's fine for me if you want to change it, as you want.

Reply:
I don't have strong feelings. If it's already done as such, keep it this way.